en/micro-integrator/docs/references/connectors/smpp-connector/smpp-connector-example.md

```diff
@@ -0,0 +1,141 @@
+# SMPP Connector Example
+
+SMPP (Short Message Peer to Peer protocol) Connector allows you to send SMS through the WSO2 EI. It uses [jsmpp API](https://jsmpp.org/) to communicate with a SMSC (Short Message service center) which is to store, forward, convert and deliver Short Message Service (SMS) messages. jsmpp is a java implementation of SMPP protocol. 
+
+## What you'll build
+
+Given below is a sample scenario that demonstrates how to work WSO2 EI SMPP Connector and send SMS messages via the SMPP protocol.
+
+The SMPP server in SMSC have all ESME (External Short Messaging Entity) addresses which is an external application that connects to a SMSC and the active connection. When you send SMS to destination, it comes to the SMSC, then one of the module in SMSC checks the destination address whether it is available or not. If it is available it create connection object that is responsible for sending SMS message.
+There are many SMPP gateways available in the world.and now almost all the Message centers support SMPP. It is not practical always to connect with real SMSC. However in this scenario we will try it with [SMSC simulator](https://github.com/smn/logica-smpp-sim).
```

Comment:
```suggestion
There are many SMPP gateways available in the world and now almost all the message centers support SMPP. It is not practical always to connect with real SMSC. However, in this scenario we will try it with [SMSC simulator](https://github.com/smn/logica-smpp-sim).
```

Reply:
Updated as following manner.
There are many SMPP gateways available in the world and now almost all the message centers support SMPP. It is not practical always to connect with real SMSC. However, in this scenario we will try it with **SMSC simulator**. Please refer the [Setting up SMPP connector](setting-up-smpp-connector.md) documentation.